apps/order/tasks.py

```diff
@@ -57,37 +101,126 @@ def send_notfication_to_publisher(order_obj, notification_type, title):
     # Send mail notifications
     # TODO: Improve subject, message, headings of email
     html_context = {
-        "heading": title,
-        "message": title
+        'heading': heading,
+        'message': mail_title,
+        'full_name': order_obj.created_by.full_name,
+        'order_data': get_order_data(order_obj)
     }
-    subject, message = title, title
+    subject, message = mail_title, mail_title
     transaction.on_commit(
         lambda: generic_email_sender.delay(
-            subject, message, recipient_list, html_context=html_context
+            subject, message, recipient_list, html_context=html_context,
+            email_for=NotificationForEnum.ORDER.value
         )
     )
     return True
 
 
-@shared_task(name="notification_sender")
+@shared_task(name='notification_sender')
 def send_notification(order_id):
     order_obj = Order.objects.get(id=order_id)
+    notification_date_attrs = {
+        'day_name': _(order_obj.order_placed_at.strftime('%A')),
+        'day': order_obj.order_placed_at.day,
+        'month_name': _(order_obj.order_placed_at.strftime('%B')),
+        'year': order_obj.order_placed_at.year,
+        'time': order_obj.order_placed_at.strftime('%H:%M'),
+        'order_code': order_obj.order_code,
```

Comment:
- Lazy text with dynamic value doesn't make sense.
- Don't use _() inside the method.

Reply:
@thenav56   day_name and month_name are passed as variables in title text. Django doesn't translated these variables, we have to pass translated text ie day_name and month_name , Do you want me to translate these variable in each if else statement ?

```
in_app_title_en = _(
                'Your order {order_code} placed on {day_name} {day}th {month_name} {year} at {time} has been recieved.'
            ).format(**notification_date_attrs)
```
Note: for day name and month name django translates automatically.

Reply:
@thenav56  as we discuss title field is removed form notification, so I deleted this part too.

Reply:
@bimal125 Sound goods.

One last thing, let's use format_lazy here instead of str.format.
https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#formatting-strings-format-lazy